projects/plugins/social/src/js/components/icon-text-section/styles.module.scss

```
@@ -0,0 +1,58 @@
+.column {
+	grid-column: 1 / -1;
+	display: grid;
+	grid-template-columns: auto 1fr;
+	grid-template-rows: repeat( 2, auto );
+	column-gap: calc( var( --spacing-base ) * 3 );
+	row-gap: calc( var( --spacing-base ) * 2 );
+
+	@supports ( display: grid ) {
+		> * + * {
+			margin-top: 0;
+		}
+	}
+
+	@media ( min-width: 600px ) {
+		grid-column: 2 / span 6;
+	}
+
+	@media ( min-width: 960px ) {
+		column-gap: calc( var( --spacing-base ) * 6 );
+		grid-column: 3 / span 8;
+	}
+}
+
+.title {
+	grid-row: 1;
+
+	@media ( min-width: 600px ) {
+		grid-column: 2;
+	}
+}
+
+.text {
+	grid-column: span 2;
+
+	a {
+		color: inherit;
+	}
+
+	@media ( min-width: 600px ) {
+		grid-column: 2;
+	}
+}
+
+.icon {
+	display: flex;
+	align-self: end;
+	grid-column: 2;
+
+	justify-self: flex-end;
+
+	margin-right: calc( var( --spacing-base ));
```

```suggestion
	margin-right: var( --spacing-base );
```

I believe `calc` is redundant here.

---

Great point ! I was playing with multiplying it by 2, but after it I forgot to remove the calc.